manifests/globals.pp

```diff
@@ -23,6 +23,10 @@
         /^(1[67].04)$/ => '7.0',
         default => '5.x',
       },
+      'Debian' => $facts['os']['release']['major'] ? {
```

Comment:
i would love to see this section refactored a little bit.
`['os']['distro']['id']` which allows a clear seperation between ubuntu and debian.

Reply:
> OSes, they are more complex than they need to be, maybe its also a good thing to clean them up later
@SimonHoenscheid do you ment this part with you comment in the issue? 
after a little more thinking this refactoring of this would be better done after merging my repo refactoring because most of the stuff then can stay in the hiera.yaml

Reply:
yes exactly. dividing Ubuntu and Debian here completely would be much more clear

Reply:
I will have a look at this later.